src/main/kotlin/com/noumenadigital/npl/cli/exception/NplCliExceptions.kt

```diff
@@ -0,0 +1,9 @@
+package com.noumenadigital.npl.cli.exception
+
+class CommandParsingException(
+    s: String,
+) : RuntimeException(s)
+
+class CommandNotFoundException(
+    s: String,
+) : RuntimeException(s)
```

Comment:
Something that I learned from Amela recently was item 72 here: https://ahdak.github.io/blog/effective-java-part-9/#:~:text=Reusing%20standard%20exceptions%20has%20several,t%20cluttered%20with%20unfamiliar%20exceptions

I don't have a strong opinion, though